packages/callout-block/src/settings.ts

```diff
@@ -24,7 +24,7 @@ export const settings: BlockSettings = {
         {
             id: 'type',
             type: 'dropdown',
-            defaultValue: Type.Warning,
+            defaultValue: Type.Info,
```

Comment:
I would also move `Info` to the top of the dropdown (also like this in the design).

Reply:
Good point. I've updated the order. Can you please approve again?
thanks